discordbot/cogs/internet.py

```diff
@@ -3,16 +3,24 @@
 """
 
 from random import randint
+from selenium import webdriver
 
+import wget
+import json
 import aiohttp
```

Comment:
If you're going to not use imports, might as well delete them. PEP8 still exists, unfortunately...